### ISSUE TITLE
Feature request: Add a globalMetadata key/value for logo alt-text

### ISSUE THREAD

**Issue by BillHertzing** [dotnet/docfx#3671]
Add a globalMetadata key/value for logo alt-text. so that both the logo path and the logo alt-text can be specified in global metadata. https://webaim.org/techniques/alttext/#logos provides guidelines for accessibility, and indicates the alt-text should be something like "Acme Company". Putting  in a global metadata key like _appLogoAltText would make the modification of accessibility text much easier than having to create a custom template

**DocFX Version Used**: 2.40.2

**Template used**: (`default` or `statictoc` or contain custom template) default

**Steps to Reproduce**: N/A

**Expected Behavior**: Replace Alt-Text for logo via globalMetadata

**Actual Behavior**: Replacement of Alt-text for logo must be done by creation of custom template and modification of  logo.tmpl.partial

**Comment by yufeih** [comment]
The `_appName` is used for logo alt by default.
